src/lib.rs

```diff
@@ -156,6 +156,51 @@ impl Vob<usize> {
         v.mask_last_block();
         v
     }
+
+    /// Create a Vob from a `u8` slice. The most significant bit of each byte comes first in the
+    /// resulting Vob.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// #[macro_use] extern crate vob;
+    /// use vob::Vob;
+    /// fn main() {
+    ///     let v = Vob::from_bytes(&[0b10100000, 0b00010010]);
+    ///     assert_eq!(v, vob![true, false, true, false, false, false, false, false,
+    ///                        false, false, false, true, false, false, true, false]);
+    /// }
+    /// ```
+    pub fn from_bytes(slice: &[u8]) -> Vob<usize> {
+        let mut v = Vob::with_capacity(slice.len()
+                                            .checked_mul(8)
+                                            .expect("Overflow detected"));
+        for i in 0..blocks_required::<usize>(slice.len() * 8) {
+            let mut w = usize::zero(); // word
+            for j in 0..bytes_per_block::<usize>() {
```

Comment:
Maybe more semantic names for `i` and `j`?

Reply:
`blk_idx`, `byte_idx`?

Reply:
Similarly below, all of the one and two-letter variable names distract from comprehension. `i` `j` `b` `k` `rb` `w` `v` -- when did you become a physicist? ;)

Reply:
I must admit, I like the short names. But, for the record, I'm far too stupid to be a physicist.